android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/activity/NavigationActivity.kt

```diff
@@ -68,6 +68,9 @@ class NavigationActivity : AppCompatActivity() {
                 enableMapLongClickIntercept = false
             }
         }
+        binding.navigationView.customizeViewBinders {
+            infoPanelEndNavigationButtonBinder = CustomInfoPanelEndNavButtonBinder(MapboxNavigationApp.current()!!)
+        }
```

Comment:
@marcotrumpet If you can make this optional and the user opts in to remove the default behavior; this will be acceptable. I think for the future, you should do a single PR per issue; this PR combined user feedback feature with cancel button handling and makes it hard to have clear separation of concerns

Reply:
The only things this custom button does other than the standard behavior is to send back the navigation_cancelled event to Dart. The standard behavior stops the navigation, mine stops the navigation and send the event that you put in the package. If I make the custom button optional, navigation will stop but no events are sent to flutter. If you want I can make it optional but whats the point if having a navigation_cancelled event if no one triggers it?